extensions/Worker.Extensions.EventGrid/src/Worker.Extensions.EventGrid.csproj

```diff
@@ -6,7 +6,8 @@
     <Description>Azure Event Grid extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>3.2.1</VersionPrefix>
+    <VersionPrefix>3.2.2</VersionPrefix>
```

Comment:
Should this be a preview still?

Reply:
Nevermind I see that it is a preview.

Reply:
So I initially changed the version to 3.3.0 but @liliankasem suggested it should be 3.2.2[ here](https://github.com/Azure/azure-functions-dotnet-worker/pull/1589#discussion_r1227228716) since we shouldn't be out of sync for major/minor version when comparing the webjobs extension.

Reply:
There's no requirement for the versions to be aligned with the WebJobs extension version. It helps avoid confusion in many cases, but semver should be followed for the worker extension and takes precedence over that convenience.

In this case, a minor version bump would be appropriate.

Reply:
Updated to 3.3.0